.github/workflows/codeforafrica-deploy-review-app.yml

```
@@ -16,7 +16,7 @@ concurrency:
 
 env:
   DOKKU_REMOTE_BRANCH: "master"
-  DOKKU_REMOTE_URL: "ssh://azureuser@ui-1.dev.codeforafrica.org"
+  DOKKU_REMOTE_URL: "ssh://azureuser@ui-2.dev.codeforafrica.org"
```

```suggestion
  DOKKU_REMOTE_URL: "ssh://azureuser@ui-1.dev.codeforafrica.org"
```
There is no `ui-2.dev` just yet